### PR TITLE
Move page subheaders under titles; minor polish to notifications

### PR DESCRIPTION
**Pull request by jamilbk**
Minor polish to notifications UI and moves the page subtitle to a more appropriate place:

# Before
Screen Shot 2022-08-04 at 1 34 42 AM

# After
Screen Shot 2022-08-04 at 1 34 07 AM

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ services:
       # doesn't match the platform under which Docker Engine is running. e.g.
       # WSL, Docker for Mac, etc.
       - /var/app/apps/fz_http/assets/node_modules
+      - /var/app/_build
```
Comment (princemaple) [inline docker-compose.yml:34]: Hmm, don't think it's necessary? It's not mounted.

File: apps/fz_http/lib/fz_http_web/live/connectivity_check_live/index_live.ex [hunk 2]
```diff
@@ -6,13 +6,21 @@ defmodule FzHttpWeb.ConnectivityCheckLive.Index do
 
   alias FzHttp.ConnectivityChecks
 
+  @page_title "WAN Connectivity Checks"
+  @page_subtitle """
+  Firezone periodically checks for WAN connectivity to the Internet and logs the result here. \
+  This is used to determine the public IP address of this server for populating the default \
+  endpoint field in device configurations.
```
Comment (princemaple) [inline apps/fz_http/lib/fz_http_web/live/connectivity_check_live/index_live.ex:13]: Most likely not needed to `\` as HTML concatenate white spaces and treat newlines as spaces anyway.

**Review by princemaple — APPROVED** [review]
Honestly I'd just write the content (title/subtitle) in the HTML.
```elixir
<%= render(FzHttpWeb.SharedView, "heading.html",
           page_title: "My title directly") %>
```
I'm happy with this though. Not too fussed either way.

File: docker-compose.yml (hunk 1 above)
Comment (jamilbk) [reply docker-compose.yml:34]: For me it was helping to prevent recompiling modules for local dev / test tasks... could tweak as necessary though.

**Comment by jamilbk** [comment]
> Honestly I'd just write the content (title/subtitle) in the HTML.
> 
> ```elixir
> <%= render(FzHttpWeb.SharedView, "heading.html",
>            page_title: "My title directly") %>
> ```
> 
> I'm happy with this though. Not too fussed either way.

Yeah, I agree. The issue is `@page_title` is a ["blessed" assign](https://hexdocs.pm/phoenix_live_view/live-layouts.html#updating-the-html-document-title) that can be access from root layouts to update `window.title` and we use it to do so.

File: apps/fz_http/lib/fz_http_web/live/connectivity_check_live/index_live.ex (hunk 2 above)
Comment (jamilbk) [reply apps/fz_http/lib/fz_http_web/live/connectivity_check_live/index_live.ex:13]: Yeah, I was a bit torn ;-)

I just defaulted to removing the newlines to keep existing behavior the same and the DOM a little easier to read.

File: docker-compose.yml (hunk 1 above)
Comment (princemaple) [reply docker-compose.yml:34]: Ah cool. I keep a container around for tests so didn't think about this. That's pretty useful.

**Pull request merged**